Changes

```diff
@@ -1,13 +1,24 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+
+3.0.0 2019-11-15
+
+ [API change]
+ - Support for Ed25519 (algorithm 15) added (#85, #51, #84)
+ - Requires libldns 1.7.0 or 1.7.1 (#85)
+
+ [Fixed]
+ - Make Travis not fail (#82)
```

Comment:
This isn't a change to the software. I don't think it belongs in this change log.

Reply:
I agree that it is a border case. I see no problem of keeping it.

Reply:
I'm not sure who you're agreeing with.

Reply:
I agree with @mattias-p

Reply:
My approach has been to include all changes to the repository, and I do not see the problem of including things concerning Travis, and in that way have it documented in the Changes file, but on your request I will exclude those. This is a change from earlier behavior, e.g. see the same file under version 2.0.0 a year ago.

I will update and remove travis from the upcoming release.